examples/milter-config.toml

```
@@ -0,0 +1,65 @@
+# Example Renews NNTP Server Configuration with Milter Support
+#
+# This configuration file demonstrates how to configure the Milter
+# protocol filter interface for external spam and content filtering.
+
+# Basic server configuration
+addr = ":119"
+site_name = "news.example.com"
+db_path = "sqlite:///var/renews/news.db"
+auth_db_path = "sqlite:///var/renews/auth.db"
+
+# Optional Milter server configuration
+# The Milter server address can be specified globally here
+[milter]
+address = "tcp://127.0.0.1:8888"        # Address of the Milter server with protocol scheme
+timeout_secs = 30                       # Connection timeout in seconds
+
+# Filter pipeline configuration
+# Filters are applied in the order specified below
+[[filters]]
+name = "HeaderFilter"               # Validates required headers (From, Subject, Newsgroups)
+
+[[filters]]
+name = "SizeFilter"                 # Validates article size limits
+
+[[filters]]
+name = "MilterFilter"               # Apply external Milter filtering
+[filters.parameters]
+address = "tcp://127.0.0.1:8888"   # Override global Milter address if needed
+timeout_secs = 30                   # Override global timeout if needed
```

Is it possible to change this to be in the form:
```suggestion
[[filters]]
name = "MilterFilter"               # Apply external Milter filtering
address = "tcp://127.0.0.1:8888"   # Milter address
timeout_secs = 30                   # Milter timeout
```